src/sdk/formatting/formatRange.test.ts

```diff
@@ -0,0 +1,13 @@
+import { formatRange } from './formatRange';
+import expect = require('expect.js');
+
+describe('formatRange', () => {
+	it('can format ranges', () => {
+		expect(formatRange('a', 'b')).to.eql('a\xa0-\xa0b');
+		expect(formatRange('100', '200')).to.eql('100\xa0-\xa0200');
+	});
+
+	it('can format with custom separator', () => {
+		expect(formatRange('a', 'b', {separator: '-'})).to.eql('a-b');
+	});
+});
```

Comment:
should we add a couple of tests for start being null, end being null, and both being null?

Reply:
Done